### PR TITLE
feat(context-menu): different options for repositories

### PR DESCRIPTION
**Pull request by alexnm**
We talked on slack about the idea of opening the repository might be misleading, so now, first two options from the context menu now open the default branch.
Opening the branch list is still possible from `See all branches`.

Closes XTD-280

Before:

image

After:

image

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7025/commits/57e22262fe6b65b7851c04a64cfb6286561cb2a3 is at https://pr7025.build.csb.dev/s/new.

File: packages/app/src/app/overmind/effects/gql/dashboard/mutations.ts [hunk 1]
```diff
@@ -335,8 +335,8 @@ export const setTeamDescription: Query<
 `;
 
 export const unmakeSandboxesTemplate: Query<
```
Comment (tristandubbeld) [inline packages/app/src/app/overmind/effects/gql/dashboard/mutations.ts:337]: Should we add a deprecation flag for these functions? Or is it not the case that they're deprecated?

File: packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/RepositoryMenu.tsx [hunk 2]
```diff
@@ -54,19 +60,28 @@ export const RepositoryMenu: React.FC<RepositoryMenuProps> = ({
...
       </MenuItem>
+
       <Menu.Divider />
+
       <MenuItem onSelect={() => window.open(branchFromDefaultUrl, '_blank')}>
         Create branch
       </MenuItem>
+      <MenuItem onSelect={() => history.push(repositoryUrl)}>
+        See all branches
+      </MenuItem>
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/RepositoryMenu.tsx:84]: Would love to see these be changed into [`MenuLink`s](https://reach.tech/menu-button/#menulink) :)

**Review by tristandubbeld — APPROVED** [review]

File: packages/app/src/app/overmind/effects/gql/dashboard/mutations.ts (hunk 1 above)
Comment (alexnm) [reply packages/app/src/app/overmind/effects/gql/dashboard/mutations.ts:337]: https://www.notion.so/codesandbox-app/Tech-debt-accounting-c4e9e7420721494faa22b47cbb35cc10?p=a9e05cf80fb147a8b71d72a3a165a0ca&pm=s

File: packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/RepositoryMenu.tsx (hunk 2 above)
Comment (alexnm) [reply packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/RepositoryMenu.tsx:84]: https://www.notion.so/codesandbox-app/Tech-debt-accounting-c4e9e7420721494faa22b47cbb35cc10?p=a9e05cf80fb147a8b71d72a3a165a0ca&pm=s

**Pull request merged**